Chetna_Mongmaw/profile/assets/styles/p3.css

```diff
@@ -0,0 +1,200 @@
+  /* body{
```

Comment:
get rid of all comments in CSS, HTML and JS files if it is not informative

Reply:
unnecessary comments removed